services.yaml

```diff
@@ -2978,6 +3052,10 @@ definitions:
       full_key:
         type: string
         example: S35817XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+      role:
```

Comment:
this is no longer a ServiceBasic as a Service shouldn't be coupled with a "user" in such a way.
We should create a new item that has both a ServiceBasic and a role as children